cmd/protoc-gen-multi/README.md

```diff
@@ -40,18 +40,17 @@ FROM --platform=$BUILDPLATFORM golang:1.21-alpine AS build
 ARG TARGETOS TARGETARCH
 
 # Add custom plugins here
-RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH \
-    go install -ldflags "-s -w" google.golang.org/protobuf/cmd/protoc-gen-go@v1.32 \
-    && go install -ldflags "-s -w" google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.3 \
-    && go install -ldflags "-s -w" github.com/planetscale/vtprotobuf/cmd/protoc-gen-go-vtproto@v0.5.0 \
-    && go install -ldflags "-s -w" github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway@v2.19
+RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go install -ldflags "-s -w" google.golang.org/protobuf/cmd/protoc-gen-go@v1.32 \
```

Comment:
Could remove some duplication by setting this first:
```dockerfile
ENV CGO_ENABLED=0 \
  GOOS=$TARGETOS \
  GOARCH=$TARGETARCH
```

Reply:
Will make another PR to test this out first (but yeah we thought about this but didn't bother changing for now).